website/database.py

```diff
@@ -30,6 +30,21 @@
 QUIZ_ANSWERS = dynamo.Table(storage, 'quizAnswers', partition_key='user', sort_key='levelAttempt')
 
 class Database:
+
+
+    def get_level(self, username):
```

Comment:
What does this mean? I think it might mean

get_highest_available_level?

Reply:
get level and adventure_index from the database